.editorconfig

```diff
@@ -0,0 +1,21 @@
+# All Files
+[*]
+charset = utf-8
+end_of_line = crlf
+indent_style = space
+indent_size = 4
+insert_final_newline = false
+trim_trailing_whitespace = true
+
+# Solution Files
+[*.sln]
+indent_style = tab
+
+# Markdown Files
+[*.md]
+trim_trailing_whitespace = false
+
+# Web Files
+[*.{htm,html,js,ts,css,scss,less}]
+insert_final_newline = true
+indent_size = 2
```

Comment:
Did you add this to make it easier to work with the ts formatting settings? If so then I approve. These settings largely match what we do always so that's fine.

Reply:
Yeah, I added that after other community members that made previous contributions say what was the standard. The default on VS is 4 spaces so that would mess all the files. Having that file makes future contributions easier as VS2017 now respect those files by default.